sources/packages/web/src/constants/award-constants.ts

```diff
@@ -53,12 +53,6 @@ export const AWARDS: AwardDetail[] = [
     description: "Canada Student Grant for Full-time Studies",
     offeringIntensity: OfferingIntensity.fullTime,
   },
-  {
```

Comment:
`CSGT` is also present here. 
![image](https://github.com/user-attachments/assets/742dd80b-374c-491e-b33d-48b77be943d4)

Do we need this in enum ?

Reply:
Removed.